contracts/Comptroller/Comptroller.sol

```diff
@@ -49,8 +49,11 @@ contract Comptroller is ComptrollerV9Storage, ComptrollerInterfaceG2, Comptrolle
     /// @notice Emitted when Venus VAI Vault rate is changed
     event NewVenusVAIVaultRate(uint oldVenusVAIVaultRate, uint newVenusVAIVaultRate);
 
-    /// @notice Emitted when a new Venus speed is calculated for a market
-    event VenusSpeedUpdated(VToken indexed vToken, uint newSpeed);
+    /// @notice Emitted when a new borrow-side VENUS speed is calculated for a market
+    event VenusBorrowSpeedUpdated(VToken indexed vToken, uint newSpeed);
```

Comment:
nit: While this says Venus it should be XVS. I suppose for consistency it is best if we keep it as XVS? We could clarify in the natspec.

Reply:
Yeah, it would be better to use XVS instead of Venus but some storage are already defined by venus. To keep the consistency I think we should keep it.

Reply:
what about  updating the natspec to say XVS?

Reply:
Done